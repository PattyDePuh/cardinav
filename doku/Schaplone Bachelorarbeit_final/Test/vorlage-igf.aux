\relax 
\providecommand\hyper@newdestlabel[2]{}
\catcode`"\active
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\select@language{ngerman}
\@writefile{toc}{\select@language{ngerman}}
\@writefile{lof}{\select@language{ngerman}}
\@writefile{lot}{\select@language{ngerman}}
\select@language{ngerman}
\@writefile{toc}{\select@language{ngerman}}
\@writefile{lof}{\select@language{ngerman}}
\@writefile{lot}{\select@language{ngerman}}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Kapitel sind ganz einfach}{3}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{loa}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}Abschnitte ebenfalls}{3}{section.1.1}}
\newlabel{eq:gleichung-eins}{{1.1}{3}{Abschnitte ebenfalls}{equation.1.1.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2}Definitionen}{3}{section.1.2}}
\newlabel{thm:fink}{{1.2}{3}{Finkscher Hauptsatz}{satz.1.2}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.1}{\ignorespaces Das ist eine Abbildung.\relax }}{4}{figure.caption.2}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:trapez}{{1.1}{4}{Das ist eine Abbildung.\relax }{figure.caption.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.3}Zweiter Abschnitt}{4}{section.1.3}}
\newlabel{sec:leichtigkeit}{{1.3}{4}{Zweiter Abschnitt}{section.1.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3.1}Tabellen}{4}{subsection.1.3.1}}
\citation{mustermann+etal-11}
\citation{bdln-lhod-02}
\citation{bdln-odgvel-05}
\citation{gj-cigtn-79}
\citation{bkns-blol-10}
\citation{bkns-blol-10}
\citation{bkns-blol-10}
\citation{bkns-blol-10}
\newlabel{fig:trapez-eins}{{1.2a}{5}{die erste Teilabbildung\relax }{figure.caption.3}{}}
\newlabel{sub@fig:trapez-eins}{{a}{5}{die erste Teilabbildung\relax }{figure.caption.3}{}}
\newlabel{fig:trapez-zwei}{{1.2b}{5}{die zweite Teilabbildung\relax }{figure.caption.3}{}}
\newlabel{sub@fig:trapez-zwei}{{b}{5}{die zweite Teilabbildung\relax }{figure.caption.3}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.2}{\ignorespaces Das ist eine Abbildung, die aus zwei Teilen besteht.\relax }}{5}{figure.caption.3}}
\newlabel{fig:zwei-trapeze}{{1.2}{5}{Das ist eine Abbildung, die aus zwei Teilen besteht.\relax }{figure.caption.3}{}}
\@writefile{lot}{\contentsline {table}{\numberline {1.1}{\ignorespaces Das ist eine Tabelle.\relax }}{5}{table.caption.4}}
\newlabel{tab:tabelle-eins}{{1.1}{5}{Das ist eine Tabelle.\relax }{table.caption.4}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3.2}Algorithmen}{5}{subsection.1.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3.3}Literaturverweise}{5}{subsection.1.3.3}}
\@writefile{lot}{\contentsline {table}{\numberline {1.2}{\ignorespaces Das ist eine Tabelle mit fester Breite.\relax }}{5}{table.caption.5}}
\newlabel{tab:tabelle-zwei}{{1.2}{5}{Das ist eine Tabelle mit fester Breite.\relax }{table.caption.5}{}}
\@writefile{loa}{\contentsline {algocf}{\numberline {1}{\ignorespaces Bin\"areSuche(Feld $A$, ganze Zahl $n$, Element $x$)\relax }}{6}{algocf.1}}
\newlabel{alg:binsearch}{{1}{6}{Algorithmen}{algocf.1}{}}
\@writefile{loa}{\contentsline {algocf}{\numberline {2}{\ignorespaces Bin\"areSucheOhneZeilennum(Feld $A$, ganze Zahl $n$, Element $x$)\relax }}{6}{algocf.2}}
\newlabel{alg:binsearchwithoutlinenumbers}{{2}{6}{Algorithmen}{algocf.2}{}}
\citation{bdln-odgvel-05}
\citation{bdln-odgvel-05}
\citation{gj-cigtn-79,bkns-blol-10}
\bibstyle{mygerapali}
\bibdata{mybib}
