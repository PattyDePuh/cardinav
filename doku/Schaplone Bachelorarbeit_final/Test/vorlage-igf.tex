\documentclass[a4paper]{scrreprt}
\usepackage[ngerman]{babel}
\usepackage[ansinew]{inputenc}

\usepackage{babelbib}
\usepackage{natbib}

\usepackage[german,linesnumbered,algoruled,longend,vlined]{algorithm2e}
\DontPrintSemicolon
\SetArgSty{}
\SetKw{KwOr}{or}
\SetKw{KwAnd}{and}
\SetKw{KwNot}{not}
\setlength{\algomargin}{3ex}

\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage{amsthm}

\usepackage{graphicx}
\usepackage{tabularx}
\usepackage[bookmarks,bookmarksnumbered]{hyperref}
\usepackage[font=small,format=hang,labelfont=bf,figurename=Abb.,tablename=Tab.]{caption}
\usepackage{subcaption}
\usepackage{enumerate}

\newtheorem{satz}{Satz}[chapter]
\newtheorem{lemma}[satz]{Lemma}
\newtheorem{beobachtung}[satz]{Beobachtung}
\newtheorem{folgerung}[satz]{Folgerung}
\newtheorem{korollar}[satz]{Korollar}
\theoremstyle{definition}
\newtheorem{definition}[satz]{Definition}
\newenvironment{beweis}{\begin{proof}}{\end{proof}}

\graphicspath{{abbildungen/}}

\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%% Bitte nur ab hier Änderungen vornehmen %%%%%%%%%%%%%%%%%%%%%

%% hier Titel und Autorennamen eintragen

\subject{Bachelorarbeit} % Geben Sie die Art der Arbeit an
\title{Titel der Arbeit} % Geben Sie hier den Titel Ihrer Arbeit an.
\author{Testvorname Testnachname} % Geben Sie Ihren Namen an. 
\date{Eingereicht am XX. YY 20ZZ} % Geben Sie das Abgabedatum an
\titlehead{Universität Osnabrück\\
Fachbereich Mathematik/Informatik\\
Institut für Geoinformatik und Fernerkundung}
\publishers{Betreuer:\\
Prof.\ Dr.-Ing.\ Jan-Henrik Haunert\\
M.Sc.\ Max Mustermann} % Geben Sie den Namen des weiteren Betreuers and
\maketitle


\tableofcontents

\chapter{Kapitel sind ganz einfach}

Fangen Sie ein Kapitel aber nicht gleich mit einem Abschnitt an,
sondern sagen Sie zuerst etwas über den Inhalt des ganzen Kapitels.
Wir werden uns also im Folgenden mit Definitionen, Sätzen, Abbildungen
und Algorithmen beschäftigen.

\section{Abschnitte ebenfalls}
Hier fängt der Text an.

Mathematische Symbole und einfache Formeln, die im Text vorkommen, werden zwischen zwei Dollarzeichen gesetzt.
So können Sie z.B.\ einfach einen Graphen  $G = (V,E)$ definieren und später im Text auf seine Eckenmenge $V$ und Kantenmenge $E$ verweisen.

Für kompliziertere Formeln können Sie dagegen die \verb+equation+-Umgebung verwenden. Beispielsweise würde die Gleichung
\begin{equation}
w = \frac{x+y}{x^2 + y^2} \label{eq:gleichung-eins}
\end{equation}
den Text unschön auseinandersperren, wenn man sie in eine Zeile zwängen würde. Wenn Sie für Gleichung~\eqref{eq:gleichung-eins} ein Label definiert haben,
können Sie auch problemlos auf sie verweisen.
Möchten Sie eine Gleichung unnummeriert lassen, können Sie die \verb+equation*+-Umgebung verwenden:
\begin{equation*}
w = \frac{x^2+y^2}{x^3 + y^3}
\end{equation*}
Allerdings sollten wichtige Formeln und insbesondere Formeln, auf die Sie später verweisen, unbedingt nummerieren.

\section{Definitionen} 
\begin{definition}
  Definitionen lassen sich leicht erstellen.
\end{definition}

Eine kurze Überleitung von einer Definition zu einem Satz hilft dem
Leser zu verstehen, wohin die Reise gehen soll.

\begin{satz}[Finkscher Hauptsatz] \label{thm:fink}
  Wichtige und grundlegende Sätze lassen sich leicht hervorheben.
\end{satz}

\begin{beweis}
  Der Satz gilt offensichtlich, denn
  \begin{equation*}
    \sum_{i=1}^{n} 1 = n
  \end{equation*}
  Zudem wird der Beweis automatisch mit einem q.e.d.-Symbol beendet.
\end{beweis}

Auf Sätze, wie z.B.\ Satz~\ref{thm:fink}, lässt sich mithilfe des
Befehls \verb+\ref{labelname}+ verweisen, wenn man in der
Satz-Umgebung einen "`Label"' mit \verb+\label{labelname}+ gesetzt
hat.  Genauso können wir auf den nächsten Abschnitt, also
Abschnitt~\ref{sec:leichtigkeit}, verweisen.  Üblicherweise beginnt
man einen Labelnamen mit dem Typ der Umgebung, auf die man verweist,
also z.B.\ \verb+\label{fig:trapez}+ für eine Abbildung (engl.\
\emph{figure}).  Ach ja, zum Hervorheben (engl.\ \emph{emphasize})
eines \emph{neuen Begriffs} verwendet man den Befehl
\verb+\emph{neuer Begriff}+, wenn der neue Begriff zum ersten Mal
verwendet wird.

\section{Zweiter Abschnitt}
\label{sec:leichtigkeit}

Auch Abbildungen, wie z.B.\ Abbildung~\ref{fig:trapez}, sind schnell
eingefügt.  Wichtig: Fügen Sie eine Abbildung immer erst nach der
ersten Referenz auf die Abbildung ein!

\begin{figure}
  \centering
  \includegraphics[page=3]{trapez}
  \caption{Das ist eine Abbildung.}
  \label{fig:trapez}
\end{figure}

Im Allgemeinen braucht man die Endung der Bilddatei beim Einbinden mit
dem Befehl \verb+\includegraphics+ nicht mit anzugeben.  Es empfiehlt
sich, alle Bilddateien in einen Unterordner abzulegen.  In mehrteilige
Abbildungen kann man mit dem Paket \verb+subcaption+ jede mit einer
eigenen Bildunterschrift versehen.  Man kann sich dann im Text sowohl
auf die Teilabbildungen (z.B.\ Abbildung~\ref{fig:trapez-eins}) als
auch auf die Gesamtabbildung (z.B.\ Abbildung~\ref{fig:zwei-trapeze})
beziehen.

Wichtig: man sollte sich im Text auf jede Abbildung wenigstens einmal
beziehen. 

\begin{figure}[h]
  \begin{subfigure}{.48\textwidth}
    \centering
    \includegraphics[page=1]{trapez}
    \caption{die erste Teilabbildung}
    \label{fig:trapez-eins}
  \end{subfigure}
  \hfill
  \begin{subfigure}{.48\textwidth}
    \centering
    \includegraphics[page=2]{trapez}
    \caption{die zweite Teilabbildung}
    \label{fig:trapez-zwei}
  \end{subfigure}
  \caption{Das ist eine Abbildung, die aus zwei Teilen besteht.}
  \label{fig:zwei-trapeze}
\end{figure}


\subsection{Tabellen}

Für beschriftete und nummerierte Tabellen sollten Sie die \verb+table+-Umgebung wählen, innerhalb der Sie die eigentliche Tabelle in die \verb+tabular+-Umgebung setzen.
Tabelle~\ref{tab:tabelle-eins} fasst die Daten von drei deutschen Städten mit fiktiven Koordinaten zusammen.

\begin{table}
\centering
\caption{Das ist eine Tabelle.}
\label{tab:tabelle-eins}
\begin{tabular}{c|c|c|c}
& Einwohnerzahl & $x$ [m] & $y$ [m] \\
\hline
Hannover & 525.000 & 11.2 & 4.6 \\
Würzburg & 133.000 & 58.1 & 1.2 \\
Osnabrück & 165.000 & 2.4 & 1.3
\end{tabular}
\end{table}

Nützlich ist auch das Paket \verb+tabularx+, mit dem Sie Tabellen mit vorgegebener Breite erzeugen können.
Beispielsweise ist Tabelle~\ref{tab:tabelle-zwei} so definiert, dass sie die ganze Seitenbreite ausfüllt.
Während die Breite der mit \verb+X+ definierten Spalte so gewählt wird, dass der vorgegebene Platz ausgefüllt ist,
wird die Breite jeder anderen Spalte an den Inhalt angepasst.

\begin{table}
\centering
\caption{Das ist eine Tabelle mit fester Breite.}
\label{tab:tabelle-zwei}
\begin{tabularx}{\linewidth}{|r|c|c|c|l|X|}
\hline
& Einwohnerzahl & $x$ [m] & $y$ [m] & Land & Oberbürgermeister \\
\hline
Hannover & 525.000 & 11.2 & 4.6 & Niedersachsen & Stefan Schostok\\
Würzburg & 133.000 & 58.1 & 1.2 & Bayern & Christian Schuchardt \\
Osnabrück & 165.000 & 2.4 & 1.3 & Niedersachsen & Wolfgang Griesert\\
\hline
\end{tabularx}
\end{table}


\subsection{Algorithmen}

Vermeiden Sie zu viele Unterebenen; das macht Ihre Arbeit nicht
unbedingt übersichtlicher.  Wir wollen hier nur zeigen, dass es mit
der \verb+algorithm+-Umgebung (aus dem Paket \verb+algorithm2e.sty+)
nicht schwer ist Algorithmen in Pseudocode zu setzen, siehe
Algorithmus~\ref{alg:binsearch}.

\begin{algorithm}
  \SetKw{True}{true}
  \SetKw{False}{false}
  \caption{BinäreSuche(Feld $A$, ganze Zahl $n$, Element $x$)}
  \label{alg:binsearch}
  \Ein{sortiertes Feld $A$, Länge $n$, gesuchtes Element $x$}
  \Aus{\True genau dann, wenn $x$ in $A$ enthalten ist}
  
  $l = 0$ \;
  $r = n-1$ \;
  
  \While{$l \le r$}{
    $m = \lfloor (l + r)/2 \rfloor$ \;
    \If{$A[m] == x$}{
      \Return \True \;
    }
    \ElseIf{$x < A[m]$}{
      $r = m - 1$ \;
    }
    \Else{
      $l = m + 1$ \;
    }
  }
  \Return \False
\end{algorithm}

Das gleiche geht problemlos auch ohne Zeilennummern, siehe
Algorithmus~\ref{alg:binsearchwithoutlinenumbers}.  Dazu benützt man
einfach in der \verb+algorithm+-Umgebung den Befehl
\verb+\LinesNotNumbered+.

\begin{algorithm}
  \LinesNotNumbered
  \SetKw{True}{true}
  \SetKw{False}{false}
  \caption{BinäreSucheOhneZeilennum(Feld $A$, ganze Zahl $n$, Element $x$)}
  \label{alg:binsearchwithoutlinenumbers}
  \Ein{sortiertes Feld $A$, Länge $n$, gesuchtes Element $x$}
  \Aus{\True genau dann, wenn $x$ in $A$ enthalten ist}
  
  $l = 0$ \;
  $r = n-1$ \;
  
  \While{$l \le r$}{
    $m = \lfloor (l + r)/2 \rfloor$ \;
    \If{$A[m] == x$}{
      \Return \True \;
    }
    \ElseIf{$x < A[m]$}{
      $r = m - 1$ \;
    }
    \Else{
      $l = m + 1$ \;
    }
  }
  \Return \False
\end{algorithm}

\subsection{Literaturverweise}

Auch Verweise auf andere Arbeiten, wie die von \citet{mustermann+etal-11}, sind ganz einfach.  Wenn es von
einem Artikel eine Konferenzversion \citep{bdln-lhod-02} und eine
Zeitschriftenversion \citep{bdln-odgvel-05} gibt, so sollten Sie stets
die Zeitschriftenversion (oder ein Buch \citep{gj-cigtn-79}) zitieren.
Tragen Sie bei einem Konferenzartikel neben Titel, Namen der Autoren
und des Tagungsbandes gegebenenfalls auch die Namen der Herausgeber in
die bib-Datei ein. ISBN-Nummern sind nicht nötig.  Seien Sie
vorsichtig mit Umlauten \citep{bkns-blol-10} in der bib-Datei.

Durch die Einbindung des \verb+natbib+-Pakets stehen Ihnen viele Befehle zur Zitierung zur Verfügung.
Sie benötigen insbesondere den Befehl \verb+\citet+, mit dem die Autorennamen außerhalb der Klammern erscheinen.
Die Autoren, z.B.\ \citet{bkns-blol-10} fungieren dann als Satzteil.
Mit dem Befehl \verb+\citep+ erscheinen der Autorennamen innerhalb der Klammern \citep{bkns-blol-10}.
Kapitel oder Theoreme, auf die Sie genau verweisen wollen, können Sie im Quelldokument in eckige Klammern setzen, z.B.\ so:
\verb+\citep[Kapitel 2]{bkns-blol-10}+.
Im Dokument erscheint dies dann als Teil des Zitats \citep[Kapitel 2]{bkns-blol-10}.

Wichtig: Zitieren Sie so, dass der Satz, in dem ein Zitat vorkommt,
auch ohne das Zitat noch Sinn ergibt; das Zitat soll eine
Zusatzinformation sein.  Also: \citet{bdln-odgvel-05}
beschäftigen sich mit der Beschriftung von Graphen.  Nicht:
\citep{bdln-odgvel-05} beschäftigen sich mit der Beschriftung von
Graphen.

Mehrere Quellen können Sie durch Komma getrennt im Quelltext angeben \citep{gj-cigtn-79, bkns-blol-10}.
Grundsätzlich sollten Sie diese Zitierweise jedoch nur wählen, wenn der Bezug zu den verschiedenen Referenzen klar ist.

\bibliographystyle{mygerapali}
\bibliography{mybib}


\end{document}
