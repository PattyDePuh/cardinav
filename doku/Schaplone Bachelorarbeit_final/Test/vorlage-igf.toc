\select@language {ngerman}
\select@language {ngerman}
\contentsline {chapter}{\numberline {1}Kapitel sind ganz einfach}{3}{chapter.1}
\contentsline {section}{\numberline {1.1}Abschnitte ebenfalls}{3}{section.1.1}
\contentsline {section}{\numberline {1.2}Definitionen}{3}{section.1.2}
\contentsline {section}{\numberline {1.3}Zweiter Abschnitt}{4}{section.1.3}
\contentsline {subsection}{\numberline {1.3.1}Tabellen}{4}{subsection.1.3.1}
\contentsline {subsection}{\numberline {1.3.2}Algorithmen}{5}{subsection.1.3.2}
\contentsline {subsection}{\numberline {1.3.3}Literaturverweise}{5}{subsection.1.3.3}
