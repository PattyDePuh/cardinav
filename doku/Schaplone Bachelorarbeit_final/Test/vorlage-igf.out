\BOOKMARK [0][-]{chapter.1}{1 Kapitel sind ganz einfach}{}% 1
\BOOKMARK [1][-]{section.1.1}{1.1 Abschnitte ebenfalls}{chapter.1}% 2
\BOOKMARK [1][-]{section.1.2}{1.2 Definitionen}{chapter.1}% 3
\BOOKMARK [1][-]{section.1.3}{1.3 Zweiter Abschnitt}{chapter.1}% 4
\BOOKMARK [2][-]{subsection.1.3.1}{1.3.1 Tabellen}{section.1.3}% 5
\BOOKMARK [2][-]{subsection.1.3.2}{1.3.2 Algorithmen}{section.1.3}% 6
\BOOKMARK [2][-]{subsection.1.3.3}{1.3.3 Literaturverweise}{section.1.3}% 7
